modules/cnv-removing-interface-from-nodes.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-removing-interface-from-nodes_{context}"]
+= Removing an interface from nodes
+
+Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object that declares the `state:` of the interface to `absent`.
+
+[NOTE]
+====
+Deleting the policy that added an interface will not change the configuration of the network policy on the node.
+Although a `NodeNetworkConfigurationPolicy` is an object in the cluster, it only represents a desired configuration.
+====
+
+.Procedure
+
+. Update the `NodeNetworkConfigurationPolicy` manifest used to create the interface. The following example removes a linux bridge:
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: <br1-eth1-policy> <1>
+spec:
+  desiredState:
+    interfaces:
+      - name: br1
+        type: linux-bridge
+        state: absent <2>
+----
+<1> Name of the policy. 
+<2> Changing the state to `absent` will remove the interfact.
+
+. Update the policy on the node and remove the interface:
++
+----
+$ oc apply -f <br1-eth1-policy.yaml> <1>
+----
+<1> File name of the policy manifest
+
+// If a policy has been applied to certain nodes using nodeSeletor, does it also have to include nodeSelector when removing the interface?
```

Comment:
There is no history kept. You create a Policy to absent `br1`, it will get removed from all nodes (or nodes matching nodeSelector if present). The proper way would be to keep the selector